resources/js/processes/screen-builder/screen.vue

```diff
@@ -372,6 +373,30 @@ export default {
           },
         ],
       },
+      {
+        id: "group_preview",
+        type: "group",
+        section: "left",
+        displayCondition: "displayPreview",
+        items: [
+          {
+            id: "button_preview_desktop",
+            type: "button",
+            title: this.$t("Preview Desktop"),
```

Comment:
Please, add the translation string to resources/lang/en.json

Reply:
It will be added